main.tf

```diff
@@ -139,7 +139,7 @@ module "gke" {
       auto_upgrade       = (var.kubernetes_channel == "UNSPECIFIED") ? false : true
       preemptible        = false
       disk_type          = "pd-standard"
-      image_type         = "COS"
+      image_type         = "cos_containerd"
```

Comment:
These are CASE sensitive. I believe the correct entry would be: COS_CONTAINERD. Remember to look at the Terraform docs for these updates. Check [here](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/e6b928288cdd79035d32ea84fd8ce6ca40979246/docs/upgrading_to_v19.0.md#-default-node-image-changed-to-cos_containerd).

Reply:
Didn't realize it was, pulled the lower case format from the Google documentation from my description and it ended up using the correct image. I'll switch this to all caps so it's more in line with the terraform-google-kubernetes-engine module usage documentation.

Reply:
Case changed, testing by deploying a v1.21.6-gke.1500 and the container runtime was still containerd